src/index.html

```diff
@@ -7,11 +7,157 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>
+    </title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="block_style">
+        <legend>Personal information</legend>
+        <label class="input_style">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="input_style">
```

Comment:
add empty line between siblings
```suggestion
        </label>
        
        <label class="input_style">
```

Reply:
thanks , is there any "must do" solutions or i can improvise?